migrations/20250304172534_add_repository_model.down.sql

```diff
@@ -0,0 +1,8 @@
+-- Add down migration script here
+CREATE TABLE repository (
+    id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL UNIQUE,
+    treeclosed INT NULL,
```

Comment:
```suggestion
    treeclosed BOOLEAN NULL,
```
Postgres supports booleans directly :)